libs/agno/agno/os/middleware/jwt.py

```diff
@@ -136,64 +328,96 @@ def _get_missing_token_error_message(self) -> str:
         else:
             return "JWT token missing"
 
-    def _is_route_excluded(self, path: str) -> bool:
-        """Check if a route path matches any of the excluded patterns."""
-        if not self.excluded_route_paths:
-            return False
+    def _create_error_response(
+        self,
+        status_code: int,
+        detail: str,
+        origin: Optional[str] = None,
+        cors_allowed_origins: Optional[List[str]] = None,
+    ) -> JSONResponse:
+        """Create an error response with CORS headers."""
+        response = JSONResponse(status_code=status_code, content={"detail": detail})
+
+        # Add CORS headers to the error response
+        if origin and self._is_origin_allowed(origin, cors_allowed_origins):
+            response.headers["Access-Control-Allow-Origin"] = origin
+            response.headers["Access-Control-Allow-Credentials"] = "true"
+            response.headers["Access-Control-Allow-Methods"] = "*"
+            response.headers["Access-Control-Allow-Headers"] = "*"
+            response.headers["Access-Control-Expose-Headers"] = "*"
+
+        return response
```

Comment:
Why is this needed? raising the `jwt` errors should already be handled by fastapi?